config

```diff
@@ -144,6 +146,22 @@ if [ "$psol_binary" = "unset" ] ; then
     psol_binary="\
         $mod_pagespeed_dir/pagespeed/automatic/pagespeed_automatic.a"
   else
+    if ! [ -d "$ngx_addon_dir/psol/lib/$buildtype" ]; then
+      echo "
+You have set --with-debug for building nginx, but precompiled Debug binaries for
+PSOL, which ngx_pagespeed depends on, aren't available.  You can either build
```

Comment:
I think I'd re-phrase the second sentence to be something like "If you're trying to debug PSOL, you must build it from source. If you just want to run nginx with verbose logging, you can use the Release binaries"